lib/nesta/plugin.rb

```diff
@@ -6,10 +6,12 @@ class << self
     self.loaded ||= []
 
     def self.register(path)
-      name = File.basename(path, '.rb')
+      # Maintain compatibility with plugins that pass filename
+      path = File.basename(path, '.rb') if path.end_with? '.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.